settings.gradle.kts

```diff
@@ -302,6 +302,7 @@ include(":instrumentation:java-http-client:library")
 include(":instrumentation:java-http-client:testing")
 include(":instrumentation:java-util-logging:javaagent")
 include(":instrumentation:java-util-logging:shaded-stub-for-instrumenting")
+include(":instrumentation:javalin:javalin-5.0:javaagent")
```

Comment:
usually we'd flatten the hierarchy to `:instrumentation:javalin-5.0:javaagent`. we add parent directory like `javalin` when there is more than 1 directory under it (there may be existing module that don't follow this).